src/sections/app.style.js

```diff
@@ -239,6 +239,14 @@ section{
     }
 }
 
+.slick-prev::before, .slick-next::before{
+  color:black !important;
```

Comment:
I am wondering if css styles can be overwritten without using the "!important" keyword. Makes it easier to manage/overwrite as app grows.

Check if you can omit !important. Okay if it can't be overwritten

Reply:
Also, preferably don't make such changes to this stylesheet. This will affect the carousel being used on all the other pages. Instead, override this style only on the page where you are rendering the carousel.

Reply:
@suhail34 this is a critical consideration. Will you give this feedback some thought?